cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ClusterCa.java

```diff
@@ -228,6 +230,7 @@ protected Map<String, CertAndKey> generateBrokerCerts(
 
             subject.addDnsName(DnsNameGenerator.podDnsName(namespace, KafkaResources.brokersServiceName(crName), node.podName()));
             subject.addDnsName(DnsNameGenerator.podDnsNameWithoutClusterDomain(namespace, KafkaResources.brokersServiceName(crName), node.podName()));
+            subject.addDnsName(node.podName());
```

Comment:
Same as above, why fo Kafka brokers need this?

Reply:
I was adding here and in CC just for consistency. I don't think it hurts. But I am fine to remove because for our specific use case, it doesn't matter at all.

Reply:
There is no consistency already. Zoo has its own set of SANs because of its weird TLS hostname verification. But we should not add that to other components as well.

Reply:
Yeah, right. I removed from the other places.